src/page-templates/content-type/content-type.tsx

```diff
@@ -424,166 +141,183 @@ const ContentTypePage: NextPage<ContentTypePageProps> = ({
+                                iconPosition="right"
+                                iconStrokeWeight="medium"
+                                iconName={ESystemIconNames.FILTER_HAMBURGER}
+                                onClick={() => setMobileFiltersOpen(true)}
+                                customWrapperStyles={{
+                                    display: ['block', null, null, 'none'],
+                                    flexBasis: ['100%', null, 'auto'],
+                                }}
+                                customStyles={{
+                                    display: ['flex', null, null, 'none'],
+                                    justifyContent: 'center',
+                                }}
+                            >
+                                Filter & Sort
+                                {!!filters.length && ` (${filters.length})`}
+                            </Button>
+
+                            <SearchResults
+                                {...resultsProps}
+                                pageNumber={pageNumber}
+                                slug={slug}
+                                updatePageMeta={updatePageMeta}
+                                contentType={contentType}
+                                extraStyles={{
+                                    order: showFeatured ? '2' : '1',
+                                }}
+                                noResultsFooter={
+                                    <Button
+                                        hasIcon={true}
+                                        iconName={ESystemIconNames.ARROW_LEFT}
+                                        iconPosition="left"
+                                        onClick={clearAll}
                                     >
-                                        {!resultIsValidating && resultData && (
-                                            <a
-                                                href={loadMoreHref}
-                                                onClick={onLoadMore}
-                                            >
-                                                <Button variant="secondary">
-                                                    Load more
-                                                </Button>
-                                            </a>
-                                        )}
-                                    </div>
-                                )}
-                            </>
-                        ) : (
-                            emptyState
-                        )}
-                    </div>
+                                        Back to all {contentType.toLowerCase()}s
+                                    </Button>
+                                }
+                            />
+                        </div>
+                    )}
+
+                    {children && children(searchProps, searchMetaProps)}
```

Comment:
This is similar to a render prop to render a completely different "body" to the page while still borrowing some of the wrappers and the searching logic. This'll be needed for the events page which has a custom multi-tiered search.